### ISSUE TITLE
Preference Changes Being Saved Without User Clicking Save

### ISSUE THREAD

**Issue by carlosmbe** [specify/specify7#3604]
**Describe the bug**
Title

**To Reproduce**
1. Go to User Tools -> Preferences
2. Change a User Interface Item. `Theme` and `Font size` are great exampls
3. Click on the logo in the navigation bar to go to the homepage.

**Expected behavior**
The changes shouldn't have been saved as I never clicked on Save. I just left. 

OR... And this is something I don't think is ideal but would make this Issue into a "Feature" Don't show the save button so that this behaviour becomes the norm of editing preferences 

**Screen Recording**
<img width="1621" alt="Screenshot 2023-06-09 at 11 13 58 AM" src="https://github.com/specify/specify7/assets/53784701/004ca6ac-7869-4a47-9b8d-7193ba6c7d61">

https://github.com/specify/specify7/assets/53784701/ab6644c2-ba79-4ea1-a527-11bf8ba25435

**Desktop:**
 - OS: Mac OSx Ventura
 - Browser: Chrome
 - Specify 7 Version: 7.8.12 and 7.8.11 Probably all of them tbh

**Comment by realVinayak** [comment]
Yeah, preferences are saved while changes are being made, and also saved if user clicks on Save. 

A design decision not a bug. Closing this for now.

**Comment by carlosmbe** [comment]
@realVinayak Why do we have a Save Button if they're already being saved? Should I move this to Enhancement then?

**Comment by realVinayak** [comment]
Well, Save Button does also save it. If you click on it, your changes will also get saved. 

Just that all the changes are also being saved while you are making changes. Yeah, Enhancement works.

**Comment by maxxxxxdlp** [comment]
Preferences are automatically saved within 5 seconds of the last edit. The save button is there anyway because:
- Clicking the save button will trigger saving right away, not in 5 seconds. Without this, if user closes the tab right away, the preference changes can get lost - if the click on save and then close, no data is lost
- Having a save button is consistent will all other pages and is what the user would expect - Chanu called this a "comfort button"
- Theoretically, we could rename the "Save" button to "Close", but then it would have to be some color other than orange, as orange is only for save buttons
